projects/step-frontend/src/lib/modules/execution/components/execution-result/execution-result.component.html

```diff
@@ -1,4 +1,3 @@
-<span [matTooltip]="displayResult" [class]="statusClass">
-  <step-icon [name]="icon" [class.spin-animation]="icon === 'refresh-cw'">
-  </step-icon>
+<span [matTooltip]="displayResult" [ngClass]="[statusClass, 'step-icon-' + result]">
```

Comment:
What is the reason of this change, if `statusClass` itself will contain 'step-icon-' + result.
Check line 33 in component's code

Reply:
Actually it contained ".step-icon-" + result that's why it did not work and I did not think it might actually contain what I needed. Fixed it now!